src/auth/computeclient.ts

```diff
@@ -60,11 +61,15 @@ export class Compute extends OAuth2Client {
    */
   protected async refreshToken(refreshToken?: string|
                                null): Promise<GetTokenResponse> {
-    const url = this.tokenUrl || Compute._GOOGLE_OAUTH2_TOKEN_URL;
+    const url =
```

Comment:
@ofrobots I really don't like this one bit.  Because of the fact that OAuth2Client lets you override the `tokenUrl`, we still have this stank in this one spot.  I see 3 options:

1. We add the ability to override the `tokenUrl` to `gcp-metadata`
2. We yank this option, and do a semver major release (doesn't have to be today)
3. We keep it around

I am very, very against options 3 🌮 What are your thoughts?